### ISSUE TITLE
Doesn't trigger input or change event

### ISSUE THREAD

**Issue by jamesjjk** [tomchentw/medium-editor-tc-mention#6]
Hey really cool component but it doesn't fire the input or change event of medium editor when having selected a mention.. Any ideas?

**Comment by tomchentw** [comment]
Uh.. interesting. I'm listening `editableInput` event and it works.

```js
medium.subscribe("editableInput", function (e) {
```

**Comment by jamesjjk** [comment]
@tomchentw I'm listening to the same event, however it is not being fired on calling selectMentionCallback

Furthermore I have noticed an error being thrown when I use a trigger as the first keyed input.. Uncaught TypeError: Cannot read property 'appendChild' of null


```
self.editor.subscribe('editableInput', (e) => {
              self._updated = true;
              console.log('trigger')
              self.change(dom.innerHTML);
          });
```

**Comment by tomchentw** [comment]
@jamesjjk you're right. I'll issue a PR.

**Comment by tomchentw** [comment]
@jamesjjk could you help to review on #7 , thanks!

**Comment by jamesjjk** [comment]
@tomchentw will check it out!

**Comment by jamesjjk** [comment]
@tomchentw Hi you didn't update the libs.. can't test the updated code in production. Furthermore I have noticed an error being thrown when I use a trigger as the first keyed input.. Uncaught TypeError: Cannot read property 'appendChild' of null

**Comment by tomchentw** [comment]
@jamesjjk I compiled `lib` and pushed it to the branch.

**Comment by jamesjjk** [comment]
@tomchentw I can confirm it works correctly in production. Thanks!

**Comment by jamesjjk** [comment]
#7 b6eadaaf87120cdf2ecc1bf4d7da6f258c808a4e

**Comment by tomchentw** [comment]
Great!

**Comment by tomchentw** [comment]
Released [v2.2.3](https://github.com/tomchentw/medium-editor-tc-mention/releases/tag/v2.2.3).
